### PR TITLE
Hidden toolkit directory in user home

### PR DESCRIPTION
**Pull request by Flanker32**
Hidden toolkit directory in user home for it may annoys users, for issue #2891

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/Activator.java [hunk 1]
```diff
@@ -89,6 +90,9 @@
 
     // User-agent header for Azure SDK calls
     public static final String USER_AGENT = "Azure Toolkit for Eclipse, v%s, machineid:%s";
+    private static final String AZURE_TOOLS_FOLDER = ".AzureToolsForEclipse";
+    private static final String AZURE_TOOLS_FOLDER_PREVIOUS = "AzureToolsForEclipse";
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/Activator.java]: AZURE_TOOLS_FOLDER_DEPRECATED

Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/Activator.java]: Fixed

File: Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/FileUtil.java [hunk 2]
```diff
@@ -28,15 +28,18 @@
...
 import java.util.zip.ZipFile;
 import java.util.zip.ZipOutputStream;
 
 public class FileUtil {
-	
-	private static final int BUFF_SIZE = 1024;
-    
+
+    private static final int BUFF_SIZE = 1024;
+    public static final String USER_HOME = "user.home";
```
Comment (andxu) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/FileUtil.java]: private

Comment (Flanker32) [reply Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/util/FileUtil.java]: Fixed

**Pull request merged**